src/plugins/unified_search/public/dataview_picker/change_dataview.test.tsx

```diff
@@ -233,44 +212,4 @@ describe('DataView component', () => {
       },
     ]);
   });
-
```

Comment:
ℹ️ the dataview picker is not visible now when in ES|QL mode so these tests are irrelevant